packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.3
+
+* Update `com.google.android.gms:play-services-maps` to 18.2.0.
```

Comment:
nit: this should be "Updates...", per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
Done.